hr_employee_benefit/models/hr_employee_benefit_rate.py

```diff
@@ -100,22 +100,24 @@ def get_all_amount_types(self):
             ('annual', _('Annual')),
         ]
 
+    @api.multi
     def _get_amounts_now(self):
         today = context_today(self)
-        self.employee_amount = self.get_amount(today)
-        self.employer_amount = self.get_amount(today, employer=True)
-
+        for rate in self:
+            rate.employee_amount = rate.get_amount(today)
+            rate.employer_amount = rate.get_amount(today, employer=True)
+    
     @api.multi
     def get_amount(self, date, employer=False):
-        self.ensure_one()
```

Comment:
@moylop260 Thanks!
This PR was only meant to be a quick bug fix. The fix also contains the removal of the constraint of being a singleton, that is not necessary IMHO. Is there any reason to still keep the _self.ensure_one()_  in the _get_amount()_ method?

I agree, that return isn't the best solution. Such refactoring is not meant to be included in this PR though.